packages/lit-dev-content/site/docs/components/rendering.md

```diff
@@ -18,6 +18,8 @@ Write your template in HTML inside a JavaScript [tagged template literal](https:
 
 Lit templates can include JavaScript _expressions_. You can use expressions to set text content, attributes, properties, and event listeners. The `render()` method can also include any JavaScript—for example, you can create local variables for use in expressions.
 
+### Renderable
```

Comment:
I think this is fine, but the canonical location for this info should be in the template docs, since different expression positions take different types. We mention the types in a similar list in *Child expressions* for example: https://lit.dev/docs/templates/expressions/#child-expressions

Putting headings there would let you link to the specific renderable you're talking about, ie, child vs attribute expression.

Reply:
makes sense. i'll update shortly

Reply:
moved the list to templating overview and linked to it from rendering

Reply:
I think we should link to https://lit.dev/docs/templates/expressions/#child-expressions specifically, because what `LitElement.render()` returns is used in a child-expression. The types are the same.

Reply:
My goal here was to create a canonical docs link for the name `renderable`, regardless of the type. IMO linking off to `child-expressions` doesn't do the trick, because it doesn't express "the thing you pass as the first arg to `render()`" as much as "a thing you can put in child position in a template"

Reply:
I think the previous setup, having Renderables (although I might call it Renderable values) in the Rendering chapter is a good thin. I fear this change reverts to the earlier state of confusion. It's hard to _find_ a list of what. you can render, and when you get to it, it's not super clear where it applies.

The rendering chapter should have a brief section that 1) clearly identifies that the render() method can return anything that can be rendered by a child expression. 2) links to a section on what you can render in a child expression for more detailed information.

My intuition is that 90% of the time people are wondering about what Lit can render, they're thinking about the return of the render() method, or what goes into a child expression. For most of the other expressions, the list is smaller and fairly intuitive.

Reply:
Rather than try and make suggestions here, I'm going to make a separate PR to compare & contrast.

Reply:
I think @arthurevans's PR looks good. The one thing it leaves out, that @bennypowers is focusing on here, is describing the first argument to lit-html's `render()`, which is slightly different from LitElement's `render()` return type in that it can include `svg` results if the container is an svg element.

Can we express that in the standalone lit-html docs?

Reply:
I'll open a ticket for that. Seems like the Standalone lit-html intro needs a face lift anyway.

Reply:
Opened as https://github.com/lit/lit.dev/issues/985